sample_tracking/app/sample-details/page.tsx

```diff
@@ -88,251 +95,43 @@ export default function SampleDetails() {
-                                        <td>{alert['detectedAt']}</td>
-                                        <td><a target="_blank" href={alert['before']['url']}><img src={alert['before']['url']} alt="Before Deforestation" height="100" width="100" /></a></td>
-                                        <td><a target="_blank" href={alert['after']['url']}><img src={alert['after']['url']} alt="After Deforestation" height="100" width="100" /></a></td>
-                                        <td><a target="_blank" href={alert['url']}><span class="material-symbols-outlined">launch</span></a></td>
-                                    </tr>
-                                ))}
-                                
-                            </tbody>
-                        </table>
-                        <div className='mapbiomas-footer'>Data from MapBiomas Alerta, 2023</div>
-                    </div>
-                : ''}
-
-                <div id='qr-code'>
-                    <div className="section-title">
-                        Sample QR code
-                    </div>
-                    <QRCodeSVG value={url} />
-                </div>
-
-            </div>
-        )
-    }
-
+    const url = `timberid.org/sample-details?trusted=${trusted}&id=${sampleId}`;
 
     return (
         <div>
-            <div className='sample-details-wrapper'>
-                <p className='title'>{selectedDoc['sample_name'] || "Sample details"}</p>
-                <div>
-                    <div className="tab-content" id="myTabContent">
-                        <DetailsTab />
+        <div className='sample-details-wrapper'>
+            <button onClick={handlePrint} className='print-button'>
+                <span className="material-symbols-outlined">print</span>
+            </button>
+            <p className='title'>{selectedDoc['sample_name'] || "Sample details"}</p>
+            <div>
+                <div className="tab-content" id="myTabContent">
+                    <div>
+                        <div className='header-validity'>
+                            <ValidityTag isTrusted={trusted === 'trusted'} city={selectedDoc['city']} lat={selectedDoc['lat']} lon={selectedDoc['lon']}/>
+                        </div>
+                        <div>
+                            {trusted != 'trusted' ? <ValiditySection selectedDoc={selectedDoc || {}} /> : ''}
+                            <SampleOverviewSection selectedDoc={selectedDoc || {}} />
+                            <SampleDetailsSection selectedDoc={selectedDoc || {}} sampleId={sampleId} />
+                            <MeasurementsSection selectedDoc={selectedDoc || {}} />
+                            {trusted != 'trusted' ? <LandUseDetailsSection selectedDoc={selectedDoc || {}} sampleId={sampleId} /> : ''}
+                            {trusted != 'trusted' ? <DeforestationAlertsSection selectedDoc={selectedDoc || {}}  /> : ''}
```

Comment:
// Integration Point #3. display your new section in the desired order with respect to other snippets.